src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
```

Comment:
Use destructuring on the `person` object.

Reply:
Don't got it? could you give me an example piece of code?

Reply:
> Don't got it? could you give me an example piece of code?

``
```suggestion
export const PersonLink: React.FC<Props> = ({ person }) => {
  const { name, sex, slug } = person;
```

Reply:
> > Don't got it? could you give me an example piece of code?
> 
> ``

Example